cmd/handler-utils.go

```diff
@@ -234,12 +234,16 @@ func extractReqParams(r *http.Request) map[string]string {
 	cred := getReqAccessCred(r, region)
 
 	// Success.
-	return map[string]string{
+	m := map[string]string{
 		"region":          region,
 		"accessKey":       cred.AccessKey,
 		"sourceIPAddress": handlers.GetSourceIP(r),
 		// Add more fields here.
 	}
+	if _, ok := r.Header[xhttp.MinIOSourceReplicationRequest]; ok {
```

Comment:
```suggestion
	if v := r.Header.Get(xhttp.MinIOSourceReplicationRequest); v != "" {
```

Perhaps is better to avoid canonical issues?

Reply:
the header value is an [empty string](https://github.com/minio/minio-go/blob/master/api-compose-object.go#L224) - hence the check for presence of header here.Otherwise minio-go would need to change

Reply:
Okay makes sense